pkg/executor/analyze.go

```diff
@@ -457,7 +438,7 @@ func (e *AnalyzeExec) handleResultsErrorWithConcurrency(
 			if isAnalyzeWorkerPanic(err) {
 				panicCnt++
 			} else {
-				logutil.Logger(ctx).Error("analyze failed", zap.Error(err))
+				logutil.BgLogger().Error("receive error when saving analyze results", zap.Error(err))
```

Comment:
What's the different between the logger and Bglogger ?

Reply:
`Logger` will try to get the current contextual logger and log some common fields from the context.

In this case, I think there is no difference.

```go
logutil.Logger(internalCtx).Info("use multiple sessions to save analyze results", zap.Int("sessionCount", sessionCount))
```

```console
[2024/08/15 15:10:14.671 +08:00] [INFO] [analyze.go:410] ["use multiple sessions to save analyze results"] [sessionCount=2]
```